x-pack/test/functional/apps/maps/index.js

```diff
@@ -42,6 +42,12 @@ export default function ({ loadTestFile, getService }) {
       loadTestFile(require.resolve('./es_geo_grid_source'));
       loadTestFile(require.resolve('./joins'));
       loadTestFile(require.resolve('./add_layer_panel'));
+      loadTestFile(require.resolve(
```

Comment:
Put an index.js file in import_geojson folder and just load the index file here. Use https://github.com/elastic/kibana/blob/master/x-pack/test/functional/apps/maps/embeddable/index.js as an example